telegram/_telegramobject.py

```diff
@@ -34,7 +34,16 @@
 
 
 class TelegramObject:
-    """Base class for most Telegram objects."""
+    """Base class for most Telegram objects.
+
+    Objects of this type a subscriptable with strings, where ``telegram_object[attribute_name]`` is
+    equivalent to ``telegram_object.attribute_name``. If the object does not have an attribute with
+    the appropriate name, a :class:`KeyError` will be raised.
```

Comment:
```suggestion
    the appropriate name, a :exc:`KeyError` will be raised.
```
I think this plays in better with intersphinx